src/config.js

```diff
@@ -6,3 +10,40 @@ export const HOSTNAME = process.env.HOSTNAME;
 export const PHOTON_SERVICE_NAME = process.env.PHOTON_SERVICE_NAME;
 export const NOMINATIM_SERVICE_NAME = process.env.NOMINATIM_SERVICE_NAME;
 export const FILE_SERVICE_NAME = process.env.FILE_SERVICE_NAME;
+
+// Load .env files in a similar way to the bikehopper-ui frontend
+// .env.development.local, .env.development, .env, etc
+//
+// We should probably only use this for development and rely on Docker
+// supplying the env vars in production
+
+const NODE_ENV = process.env.NODE_ENV;
+if (!NODE_ENV) {
+  throw new Error(
+    'The NODE_ENV environment variable is required but was not specified.'
+  );
+}
+
+const dotenvPath = '.env';
+
+// https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
+const dotenvFiles = [
+  `${dotenvPath}.${NODE_ENV}.local`,
+  // Don't include `.env.local` for `test` environment
+  // since normally you expect tests to produce the same
+  // results for everyone
+  NODE_ENV !== 'test' && `${dotenvPath}.local`,
+  `${dotenvPath}.${NODE_ENV}`,
+  dotenvPath,
+].filter(Boolean);
```

Comment:
can you change this `.filter(Boolean)` to something more idiomatic? It looks like it filters all Boolean values but it actually filters all false-y values.